usr/local/www/status_ntpd.php

```diff
@@ -163,114 +162,146 @@
 
 $pgtitle = array(gettext("Status"),gettext("NTP"));
 $shortcut_section = "ntp";
+
+function print_cell($contents) {
```

Comment:
Since there is no logic involved I fail to see the benefit of this construction over simply

```php
<td><?=$server['delay']?></td>
```